internal/api/web/handlers/quiz.go

```diff
@@ -12,13 +15,42 @@ import (
 func RegisterQuizHandlers(e *echo.Echo) {
 	e.GET("/", quizHomePage)
 	e.GET("/quizpage", GetQuizPage)
+	e.GET("/nextquestion", GetIsCorrect)
+	e.POST("/nextquestion", POSTNextQuestion)
```

Comment:
Consider Making the client ask the server for a specific next question with an id instead of a general "nextquestion". As we discussed.